WidgetKit/QuickLink.swift

```diff
@@ -0,0 +1,95 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import SwiftUI
+import Shared
+
+enum QuickLink: Int {
+    case search = 1
+    case copiedLink
+    case privateSearch
+    case closePrivateTabs
+
+    public var imageName: String {
+        switch self {
+        case .search:
+            return "faviconFox"
+        case .privateSearch:
+            return "smallPrivateMask"
+        case .copiedLink:
+            return "copy_link_icon"
+        case .closePrivateTabs:
+            return "delete"
+        }
+    }
+
+    public var headlineTxt: String {
+        return String(String.newSearchButtonLabel.split(separator: " ")[2])
```

Comment:
@noorhashem whats the purpose of separating headline text?

Reply:
@nbhasin2  this is for the separate "Firefox" large string in new search and new private search buttons in the latest design updates.